src/ui/public/vislib/visualizations/_map.js

```diff
@@ -259,16 +259,23 @@ export default function MapFactory(Private, tilemap, $sanitize) {
         if (NWlng < -180) {
           NWlng += 360;
         }
+
+        const SELat = bounds.getSouthEast().lat;
+        const NWLat = bounds.getNorthWest().lat;
```

Comment:
I ended up switching the bbox object members instead, so both the order of the long and lat declarations all line up.